gwctl/pkg/printer/gatewayclasses.go

```diff
@@ -89,11 +100,34 @@ func (gcp *GatewayClassesPrinter) PrintDescribeView(resourceModel *resourcedisco
 	index := 0
 	for _, gatewayClassNode := range resourceModel.GatewayClasses {
 		index++
+		apiVersion, kind := gatewayClassNode.GatewayClass.GetObjectKind().GroupVersionKind().ToAPIVersionAndKind()
+		metadata := gatewayClassNode.GatewayClass.GetObjectMeta()
 
+		// views ordered with respect to https://deploy-preview-2723--kubernetes-sigs-gateway-api.netlify.app/geps/gep-2722/#:~:text=gwctl%20describe%20gatewayclass%20foo%2Dcom%2Dexternal%2Dgateway%2Dclass
```

Comment:
nit: You can either remove this comment, or change the link to a permanent one at https://gateway-api.sigs.k8s.io/geps/gep-2722/